drawee/src/main/java/com/facebook/drawee/controller/AbstractDraweeController.java

```diff
@@ -532,6 +534,10 @@ private void onNewResultInternal(
           mDataSource = null;
           mSettableDraweeHierarchy.setImage(drawable, 1f, wasImmediate);
           getControllerListener().onFinalImageSet(id, getImageInfo(image), getAnimatable());
+        } else if (deliverTempResult) {
+          logMessageAndImage("set_temporary_result @ onNewResult", image);
+          mSettableDraweeHierarchy.setImage(drawable, 1f, wasImmediate);
+          getControllerListener().onFinalImageSet(id, getImageInfo(image), getAnimatable());
           // IMPORTANT: do not execute any instance-specific code after this point
```

Comment:
This message should also be here for the original block. I'll just add it when I'm importing the change.